### PR TITLE
De only/v1

### PR DESCRIPTION
**Pull request by victorjulien**
https://redmine.openinfosecfoundation.org/issues/7414
https://redmine.openinfosecfoundation.org/issues/7433

SV_BRANCH=https://github.com/OISF/suricata-verify/pull/2159

### CODE REVIEW

File: src/output-json-alert.c [hunk 1]
```diff
@@ -791,15 +791,37 @@ static int AlertJsonDecoderEvent(ThreadVars *tv, JsonAlertLogThread *aft, const
...
+        AlertJsonHeader(p, pa, jb, json_output_ctx->flags, NULL, NULL);
+
+        if (PacketIsTunnel(p)) {
+            AlertJsonTunnel(p, jb);
         }
 
-        /* just the timestamp, no tuple */
-        jb_set_string(jb, "timestamp", timebuf);
+        /* payload */
+        if (json_output_ctx->flags &
```
Comment (victorjulien) [inline src/output-json-alert.c:806]: this seems  useless, as we won't have a payload for this case?

Comment (inashivb) [reply src/output-json-alert.c:806]: Not even for the `decode-event:ipv6.frag_pkt_too_large` kind?

Comment (victorjulien) [reply src/output-json-alert.c:806]: I think that would put us in the first condition here?
```c
    if (PacketIsIPv4(p) || PacketIsIPv6(p)) { 
        return AlertJson(tv, aft, p);
    } else if (p->alerts.cnt > 0) { 
        return AlertJsonDecoderEvent(tv, aft, p);
    }
```

**Comment by suricata-qa** [comment]
Information: QA ran without warnings. 

Pipeline 23665

**Comment by victorjulien** [comment]
Replaced by #12221.

**Pull request closed without merging**